docs/Guides/V2Upgrade.md

```diff
@@ -14,18 +14,14 @@ In the [SDK release](https://github.com/bcnmy/biconomy-client-sdk/releases/tag/r
 
 ### Steps to upgrade
 
-1. create a Biconomy account V1 instance and execute **`getUpdateImplementationData`** transaction.
+1. create a Biconomy account V1 instance and execute **`updateImplementationUserOp`** transaction. This includes upgrade transaction and the enabling ECDSA validation module.
 
 ```tsx
   const biconomyAccount = new BiconomySmartAccount(biconomySmartAccountConfig);
   const biconomySmartAccount = await biconomyAccount.init();
 
-  const tx = await biconomySmartAccount.getUpdateImplementationData() 
-  console.log('account update tx', tx)
-
-  // transaction is any existing transaction
-
-  let partialUserOp = await biconomySmartAccount.buildUserOp([transaction, tx]);
+  const partialUserOp = await biconomySmartAccount.updateImplementationUserOp() 
```

Comment:
will have to also point out that this gives you partial userop without paymasterAndData. if you want to use a paymaster then after this step they have to make a separate call to the paymaster to getPaymasterAndData (this might need some policies to be updated as well like on the account address updateImplementation method and setupAndEnableModule method)